tests/integration/hive_metastore/test_workflows.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+from dataclasses import replace
+from datetime import timedelta
+
+import pytest
+from databricks.labs.blueprint.installer import RawState
+from databricks.sdk.errors import NotFound
+from databricks.sdk.retries import retried
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+@pytest.mark.parametrize(
+    "prepare_tables_for_migration,workflow",
+    [
+        ("regular", "migrate-tables"),
+        ("hiveserde", "migrate-external-hiveserde-tables-in-place-experimental"),
```

Comment:
With the changes to `table_migration.py` this test passes for all workflows. However, for the hiveserde workflows, the test already passed before the changes, which surprises me

Reply:
This happened because the table index is set in the migrate view, but if no views are migrated it is not reset